torch/_inductor/codegen/triton.py

```diff
@@ -1062,7 +1062,7 @@ def exp(x):
         more details.
         """
         if config.use_fast_math:
```

Comment:
```suggestion
        if not config.use_fast_math:
```

I think these options are swapped (this is likely the root cause of the bug).

Reply:
yes agree